app/forms/Countries.scala

```diff
@@ -31,13 +31,28 @@ object Countries {
       .map(_.replace("\"", ""))
       .toList
 
+  val countriesThreeDigitMap : Map[String, String] = {
+    val filename = "conf/three-digit-country-code-to-country-name.csv"
+    val lines = Source.fromFile(filename).getLines.toList
+    lines.foldLeft(List[(String, String)]()) {
+      (acc, item) =>
+        val lineArray = item.split("=")
+        val (countryCode, countryName) = (lineArray.head, lineArray.last)
```

Comment:
.head and .last could throw exception here if the list is empty (very unlikely but i think we should handle it just incase)

Reply:
Fixed